services/src/contexts/migrations/database_migration.rs

```diff
@@ -70,30 +70,32 @@ pub enum MigrationResult {
 /// # Panics
 ///
 /// Panics if there is an error in the migration logic.
+///
 pub async fn migrate_database<Tls>(
-    conn: &mut PooledConnection<'_, PostgresConnectionManager<Tls>>,
+    connection: &mut PooledConnection<'_, PostgresConnectionManager<Tls>>,
     migrations: &[Box<dyn Migration>],
-    prefix: Option<&str>,
+    migration_if_uninitialized: Option<Box<dyn Migration>>,
```

Comment:
I feel like this shouldn't be a migration. Why should this be a boxed option? There should be exactly one initialization logic to be applied when the schema is uninitialized.

Reply:
I designed it this way because you designed it to get a set of migrations as input.
Already now, you are required to input a list of migration. If you skip the first one, it is not applied.

Reply:
[eaf44d5](https://github.com/geo-engine/geoengine/pull/945/commits/eaf44d5dbb00383dc5c00149813d0f445adcb75e)